src/slack/slack.js

```diff
@@ -655,9 +658,20 @@ export class Slack {
     if (action === 'office') {
       order.isHome = actionValue === HOME_VALUE
       order.office = order.isHome ? HOME_TO_OFFICE[order.country] : actionValue
+      await updateMessage({
+        actions: ORDER_SPINOFF_ACTIONS,
+        fields: [...this.getOrderFields(order), {title: 'Select your spinoff?'}],
+      })
+
+      return false
+    }
+
+    // office/home
```

Comment:
I have these comments to describe the path user took to get there. here it should be:
```suggestion
    // ?-spinoff
```
because in 1st step the user chose home or office, in 2nd he chose the spinoff.
it also makes sense to update all next comments below to factor the spinoff in - add it as the 2nd step.